libs/algo/odc/algo/_geomedian.py

```diff
@@ -9,6 +9,7 @@
 from ._dask import randomize, reshape_yxbt
 from ._masking import to_float_np, from_float_np
 from ._memsink import yxbt_sink
+from .backend import rust_geomedian
 
```

Comment:
backend will need to be selectable, at least at first to allow easier comparison both in terms of equivalence and performance.

Reply:
ah yep, good idea